beacon-chain/sync/metrics.go

```diff
@@ -86,7 +86,7 @@ var (
 		prometheus.HistogramOpts{
 			Name:    "block_arrival_latency_milliseconds",
 			Help:    "Captures blocks propagation time. Blocks arrival in milliseconds distribution",
-			Buckets: []float64{250, 500, 1000, 1500, 2000, 4000, 8000, 16000},
+			Buckets: []float64{100, 250, 500, 750, 1000, 4000, 8000, 12000},
```

Comment:
Please add back 1500 and 2000.

Reply:
Ops. I had added it but forgot to push it. Done now